tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -8015,6 +8015,24 @@ def err_hlsl_hitobject_unsupported_stage : Error<
    "dx::HitObject is unavailable in shader stage '%0' (requires 'raygeneration', 'closesthit' or 'miss')">;
 // HLSL Change Ends
 
+// Linear Algebra Operations
```

Comment:
eeps, set column width to 80. Note to self.

Reply:
ok to resolve?